tests/benchq/problem_ingestion/test_molecule_instance_generation.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from benchq.problem_ingestion import (
+    WATER_MOLECULE,
+    ChemistryApplicationInstance,
+    generate_hydrogen_chain_instance,
+)
+
+
+@pytest.mark.parametrize(
```

Comment:
Tests look a bit sparse – I'm not sure what exactly should be included here, but it looks to me that some of the input arguments to the `ChemistryApplicationInstance` are not covered by tests at all, e.g.: AVAS ones.

Reply:
The AVAS arguments were actually covered by the water molecule test. But I refactored it to have the test explicitly set the AVAS options in 9f2cc32c779118dbccfe6fa4e6bf20a9082a12a0 and also added tests for cases that are invalid.